lib/form1010_ezr/veteran_enrollment_system/associations/reconciler.rb

```diff
@@ -0,0 +1,139 @@
+# frozen_string_literal: true
```

Comment:
I created this `Reconciler` class because the logic is, in my opinion, worth testing.